packaging/windows/shotcut.nsi

```diff
@@ -132,6 +141,8 @@ Section "Uninstall"
   ; Remove registry keys
   DeleteRegKey HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Shotcut"
   DeleteRegKey HKLM SOFTWARE\Shotcut
+  DeleteRegStr HKCR ".mlt"
+  DeleteRegStr HKCR "Shotcut\shell\open\command"

```

Comment:
Also, `DeleteRegStr HKCR "Shotcut\shell\open\command"`
should be `DeleteRegKey HKCR "Shotcut"` to delete the whole subtree we created.